pkg/reconcilermanager/controllers/reconciler_base.go

```diff
@@ -820,6 +849,40 @@ func (r *reconcilerBase) patchSyncMetadata(ctx context.Context, rs client.Object
 	if err != nil {
 		return true, fmt.Errorf("Sync metadata update failed: %w", err)
 	}
-	r.logger(ctx).Info("Sync metadata update successful")
+	r.Logger(ctx).V(3).Info("Patching object metadata successful",
+		logFieldObjectRef, syncKey.String(),
+		logFieldObjectKind, r.syncKind)
 	return true, nil
 }
+
+func (r *reconcilerBase) requeueRSyncRequest(ctx context.Context, obj client.Object, rsRef types.NamespacedName) []reconcile.Request {
```

Comment:
nit: this method and `isAnnotationValueTrue` seem like fine refactors, but would simplify review if they were separate changes

Reply:
The refactor allows the logging i think, but I'll pull out the refactor and we can merge that first if you like

Reply:
https://github.com/GoogleContainerTools/kpt-config-sync/pull/1563 merged. rebased here.